doc/specs/#2871 - Pane Navigation/#2871 - Pane Navigation.md

```diff
@@ -0,0 +1,298 @@
+  - This now forks the "Direction" enum into "MoveFocusDirection" and
+    "ResizeDirection" (because `resizePane(last)` doesn't make any sense).
+
+This proposal doesn't really have any special consideration for the pane
+switcher UX. Neither of these actions would summon the pane switcher UX.
+
+### Proposal E: Three actions
+
+* `focusPane(target=id)`
+* `moveFocus(direction="up|down|left|right")`
+* `focusLastPane(usePaneSwitcher=false|true)`
+
+In this design, neither `focusPane` nor `moveFocus` will summon the pane
+switcher UI (even once it's added). However, the `focusLastPane` one _could_,
+and subsequent keypresses could pop you through the MRU stack, while it's
+visible? The pane switcher could then display the panes for the tab in MRU
+order, and the user could just use the arrow keys to navigate the list if they
+so choose.
+
+* **Pros**:
+  - Each action does explicitly one thing.
+  - Design accounts for future pane switcher UX
+* **Cons**:
+  - Three separate actions for similar behavior
+
+### Proposal F: It's literally just tmux
+
+_Also known as the "one action to rule them all" proposal_
+
+`focusPane(target=id, direction="up|down|left|right|last")`
+
+Previously, this design was avoided, because what does `focusPane(target=4,
+direction=down)` do? Does it focus pane 4, or does it move focus down?
+
+`tmux` solves this in one action by just doing both!
+
+```
+Make pane target-pane the active pane ...  If one of -D, -L, -R, or -U is used,
+respectively the pane below, to the left, to the right, or above the target pane
+is used.
+```
+_from `man tmux`_.
+
+So `focusPane(target=1, direction=up)` will attempt to focus the pane above pane
+1. This action would not summon the pane switcher UX, even for
+`focusPane(direction=last)`
+
+* **Pros**:
+  - Fewest redundant actions
+* **Cons**:
+  - Is this intuitive? That combining the params would do both, with `target`
+    happening "first"?
+  - Assumes that there will be a separate action added in the future for "Open
+    the pane switcher (with some given ordering)"
+
+
+> 👉 **NOTE**: At this point, the author considered "Do we even want a separate
+> action to engage the tab switcher with panes expanded?" Perhaps panes being
+> visible in the tab switcher is just part fo the tab switcher's behavior. Maybe
+> there shouldn't be a separate "open the tab switcher with the panes expanded
+> to the pane I'm currently on, and the panes listed in MRU order" action.
+
+## Conclusion
+
+After much discussion as a team, we decided that **Proposal D** would be the
+best option. We felt that there wasn't a need to add any extra configuration to
+invoke the "pane switcher" as anything different than the "tab switcher". The
+"pane switcher" should really just exist as a part of the functionality of the
+advanced tab switcher, not as it's own thing.
+
+Additionally, we concurred that the new "direction" value should be `prev`, not
+`last`, for consistency's sake.

```

Comment:
nit: you _could_ just update Proposal D to use `prev` instead of `last`. But meh.